pyrdp/player/LiveEventHandler.py

```diff
@@ -80,6 +90,46 @@ def onListDirectory(self, deviceID: int, path: str):
         request = PlayerDirectoryListingRequestPDU(self.layer.getCurrentTimeStamp(), deviceID, path)
         self.layer.sendPDU(request)
 
+    def addToDownloadQueue(self, item: Union[File, Directory], targetPath: str, dialog: FileDownloadDialog):
+        job = (item, targetPath, dialog)
+
+        self.jobsQueue.add(job)
+
+        if self.currentDownload == None:
+            self.dispatchDownload()
+
+    def dispatchDownload(self):
+        # We download each file of a directory before enumerating other directories.
+        # Otherwise we may fill the stack with too many queued items.
+        # When we're done with both file and directories, we start an other queued job
+
+        # Request download of a queued file
+        if len(self.fileDownloadQueue) != 0:
```

Comment:
`myQueue.empty()` exists I think

Reply:
(see elifs as well)

Reply:
It doesn't, in python `if not myQueue` is falsy if the queue is empty. Even though its very pythonic, it's not very verbose. I think using `len(myQueue) != 0` should stay, as seen everywhere else in the code base

Reply:
im confused: https://docs.python.org/3/library/queue.html#queue.Queue.empty

Reply:
Thats the definition of the queue class, which is not what I'm using. The Queue class is meant for multi-threaded application.

Reply:
Damn, I overlooked that! I understand.